openlibrary/macros/BookByline.html

```diff
@@ -1,21 +1,26 @@
-$def with(author_names_and_urls, limit=None, overflow_url=None, attrs='')
+$def with(author_names_url_birth_date_death_date, limit=None, overflow_url=None, attrs='')
```

Comment:
This tuple is becoming a little large! I guess we should update this to pass in a dict at this point. So a dict with 4 fields, the author name, author url, birth date, death date.